controller.js

```diff
@@ -1,7 +1,89 @@
-var Calculator = require( './Calculator' )
-var clickHandler = document.getElementById( 'calculator-keypad' )
+var Calc = new Calculator() //eslint-disable-line
+var clickElement = document.querySelector( '.calculator-keypad' )
 
-clickHandler.addEventListener( '.click', function( event ) {
-  var value = event.target
-  console.log( event.target )
-} )
+clickElement.addEventListener( 'click', calculatorHandler.bind( this ) )
+document.addEventListener( 'keypress', calculatorHandler.bind( this ) )
```

Comment:
I'm not sure why you have `.bind( this )` on these callbacks. The this you are binding to is the window / global object. There could be some confusion here.